src/store/modules/product/actions.ts

```diff
@@ -93,6 +93,79 @@ const actions: ActionTree<ProductState, RootState> = {
     }
   },
 
+  // Update QueryString
+  async updateQueryString({ commit }, queryString) {
```

Comment:
I think this is not needed. We could use updateProductFilters

Reply:
Done Sir.